packages/pages/src/upgrade/pagesUpdater.ts

```diff
@@ -6,13 +6,13 @@ import { execSync } from "child_process";
 import { readJsonSync } from "./migrateConfig.js";
 import { fileURLToPath } from "url";
 
-const pagesSlashComponentsRegex = /"@yext\/pages\/components"/g;
-const sitesComponentsRegex = /"@yext\/sites-components"/g;
-const sitesComponentsReplacement = '"@yext/sites-components"';
-const pagesComponentsReplacement = '"@yext/pages-components"';
-const reactComponentsRegex = /"@yext\/react-components"/g;
-const fetchImportRegex = /\nimport { fetch } from "@yext\/pages\/util";/g;
-const markdownRegex = 'Markdown.{1,10}from "@yext\\/react-components';
+const pagesSlashComponentsRegex = /@yext\/pages\/components/g;
+const sitesComponentsRegex = /@yext\/sites-components/g;
+const reactComponentsRegex = /@yext\/react-components/g;
+const fetchImportRegex = /\nimport { fetch } from ["']@yext\/pages\/util["'];/g;
```

Comment:
Not from this change, but this doesn't catch no spaces around `fetch`

`import {fetch} from '@yext/pages/util';`

or more than one import

`import { fetch, foo } from '@yext/pages/util';`

also what's the \n for? I think that shouldn't be there.

Reply:
Without the \n it wasn't clearing the line properly

I can look into making that check smarter

This works on the starter repo

Reply:
What do you mean by _this_? All 3 cases? I was trying in regex101.com

Reply:
The check that is there currently works for fetch as it appears on the starter repo.

I just updated the PR to cover all cases however